lgc/patch/Gfx6ConfigBuilder.cpp

```diff
@@ -492,8 +492,8 @@ template <typename T> void ConfigBuilder::buildVsRegConfig(ShaderStage shaderSta
 
     // Set fields CLIP_DIST_ENA_0 ~ CLIP_DIST_ENA_7 and CULL_DIST_ENA_0 ~ CULL_DIST_ENA_7
     unsigned paClVsOutCntl = GET_REG(&config->vsRegs, PA_CL_VS_OUT_CNTL);
-    paClVsOutCntl |= clipDistanceMask;
-    paClVsOutCntl |= (cullDistanceMask << 8);
+    unsigned finalMask = clipDistanceMask | cullDistanceMask;
```

Comment:
```suggestion
    unsigned finalMask = clipDistanceMask | cullDistanceMask;
    // NOTE: <Explain why we should modify cullDistanceMask by taking clipDistanceMask into account>
    cullDistanceMask = clipDistanceMask | cullDistanceMask;
    finalClipCullMask = clipDistanceMask | (cullDistanceMask << 8);
```

Reply:
Fixed.

Reply:
Do you happen to know if there's a performance implication to setting more bits in this mask?

I find it difficult to understand the comment. How about:

> Note: Point primitives are only affected by the cull mask, so enable culling also based on clip distances.

Reply:
Ok, I have updated to the new comments in the latest commit.